app/src/main/java/org/simple/clinic/facility/FacilityPayload.kt

```diff
@@ -89,7 +92,8 @@ data class FacilityPayload(
         updatedAt = updatedAt,
         syncStatus = syncStatus,
         deletedAt = deletedAt,
-        config = config ?: FacilityConfig(diabetesManagementEnabled = false)
+        teleconsultationPhoneNumber = "",
```

Comment:
This could be:

```kotlin
teleconsultationPhoneNumber = teleconsultationPhoneNumber ?: ""
```

so that we won't have to change this again when the API starts sending the field.